.travis.yml

```diff
@@ -1,15 +1,12 @@
 language: python
-sudo: false
+dist: xenial
 python:
     - 2.7
-    - 3.4
     - 3.5
     - 3.6
-    # Force a newer PyPy on the old 'precise' CI image.
-    # After September 2017 this should be the default and the version
-    # pin can be removed.
-    - pypy-5.6.0
-    - pypy3.5-5.8.0
+    - 3.7
+    - pypy2.7-5.0.0
```

Comment:
This version doesn't seem to be installable. Typo?

Reply:
Typo.